From 768856a34fa5bcf696d7b13daae9785370e6fc28 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 9 Jun 2014 11:59:19 -0400
Subject: [block] blk-mq: remove newly added instances of __cpuinit

Message-id: <1402315159-3749-7-git-send-email-prarit@redhat.com>
Patchwork-id: 83103
O-Subject: [RHEL7.1 PATCH BZ 1105893 12/12] blk-mq: remove newly added instances of __cpuinit
Bugzilla: 1105893
RH-Acked-by: Rafael Aquini <aquini@redhat.com>
RH-Acked-by: Lenny Szubowicz <lszubowi@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Vivek Goyal <vgoyal@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1105893

commit f618ef7c47934d1686a764d0c9f70f23e566683f
Author: Paul Gortmaker <paul.gortmaker@windriver.com>
Date:   Thu Nov 14 08:26:02 2013 -0700

    blk-mq: remove newly added instances of __cpuinit

    The new blk-mq code added new instances of __cpuinit usage.
    We removed this a couple versions ago; we now want to remove
    the compat no-op stubs.  Introducing new users is not what
    we want to see at this point in time, as it will break once
    the stubs are gone.

    Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
    Signed-off-by: Jens Axboe <axboe@kernel.dk>

Cc: Jeff Moyer <jmoyer@redhat.com>
Cc: Vivek Goyal <vgoyal@redhat.com>
Cc: Tony Camuso <tcamuso@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/block/blk-mq-cpu.c b/block/blk-mq-cpu.c
index 58f2791..3146bef 100644
--- a/block/blk-mq-cpu.c
+++ b/block/blk-mq-cpu.c
@@ -13,8 +13,8 @@
 static LIST_HEAD(blk_mq_cpu_notify_list);
 static DEFINE_SPINLOCK(blk_mq_cpu_notify_lock);
 
-static int __cpuinit blk_mq_main_cpu_notify(struct notifier_block *self,
-         unsigned long action, void *hcpu)
+static int blk_mq_main_cpu_notify(struct notifier_block *self,
+      unsigned long action, void *hcpu)
 {
  unsigned int cpu = (unsigned long) hcpu;
  struct blk_mq_cpu_notifier *notify;
diff --git a/block/blk-mq.c b/block/blk-mq.c
index 32c41cf..1b8b50d 100644
--- a/block/blk-mq.c
+++ b/block/blk-mq.c
@@ -1389,7 +1389,7 @@ void blk_mq_free_queue(struct request_queue *q)
 }
 
 /* Basically redo blk_mq_init_queue with queue frozen */
-static void __cpuinit blk_mq_queue_reinit(struct request_queue *q)
+static void blk_mq_queue_reinit(struct request_queue *q)
 {
  blk_mq_freeze_queue(q);
 
@@ -1406,8 +1406,8 @@ static void __cpuinit blk_mq_queue_reinit(struct request_queue *q)
  blk_mq_unfreeze_queue(q);
 }
 
-static int __cpuinit blk_mq_queue_reinit_notify(struct notifier_block *nb,
-  unsigned long action, void *hcpu)
+static int blk_mq_queue_reinit_notify(struct notifier_block *nb,
+          unsigned long action, void *hcpu)
 {
  struct request_queue *q;
 
-- 
1.7.1